ewlegal-opinion.sty

```diff
@@ -1,23 +1,43 @@
-\ProvidesPackage{engwales-opinion}[2021-02-13 Package for pleading creation in England/Wales styles]
+%% TODO Stock beginning text, copyrights and project info and all that
 
-\RequirePackage[explicit]{titlesec}
-\RequirePackage{enumitem}
-\RequirePackage{fancyhdr}
+\ProvidesPackage{ewlegal-opinion}[2021-02-13 England and Wales Opinion Legal Template]
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Typeset Elements
+%%%%%
 
 %% Formatting the main title
 \renewcommand\maketitle{%
   \begin{center}
     \bfseries
-    \renewcommand\arrayrulewidth{1.1pt}
-    \begin{tabular}{@{\hspace{12pt}}c@{\hspace{12pt}}}
-      \hline\\[-12pt]
-      \MakeUppercase{\@title}\\\ \\[-12pt]
-      \hline
-    \end{tabular}
+    \tramlines{\@title}
   \end{center}
   \medskip
 }
 
+%% Determine the spacing of the numbers in the point env
+% \renewcommand{\pointspacing}{leftmargin=0,labelsep=1cm}
+% TODO This work around to reusing the point env does not work. FIX!
+
+%% Create intro and facts commands for supposed convience
+\newcommand{\intro}{\section{INTRODUCTION}}
+\newcommand{\facts}{\section{FACTS}}
+
+%% Provides a signature command
+\newcommand{\signature}
+{
+  \vspace*{3em}\flushright%
+  \@author\\\@date
+}
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Typesetting
+%%%%%
+
 %% Formatting section and subsection titles
 \titleformat{\section}{\normalfont\bfseries}{\thesection}{1em}
   {\vspace*{-0.25em}\underline{\MakeUppercase{#1}}\medskip}
```

Comment:
i'm not sure whether we should help users uppercase, especially with @passamo9's [thumbs up on small-cased titles](https://github.com/passamo9/LaTeX_EnglandWales_Templates/pull/31#discussion_r579626859). plus `\underline` does not work well with multiline `\MakeUppercase`s, altho that probably won't matter to section headings.

Reply:
Yeah, I just kept it the same from what it was. I made it uppercase to mimic how INTRODUCTION and FACTS are written, but I do agree with your points, though multi-line sections are probably not a worry, but we'll see what passamo9 says.